tools/bootstrap_tests.py

```diff
@@ -50,6 +50,7 @@
                                 ('/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=barisits/CN=692443/CN=Martin Barisits', 'x509', 'martin.barisits@cern.ch'),
                                 ('/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=tbeerman/CN=722011/CN=Thomas Beermann', 'x509', 'thomas.beermann@cern.ch'),
                                 ('/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=ruciobuildbot/CN=692443/CN=Robot: Rucio build bot', 'x509', 'rucio.build.bot@cern.ch'),
+                                ('CN=Stefan Prenner,CN=822876,CN=sprenner,OU=Users,OU=Organic Units,DC=cern,DC=ch', 'x509', 'stefan.prenner@cern.ch'),
```

Comment:
this doesn't look right?

Reply:
It works, at least on his dev machine. If ordered in the 'right' way the server does not accept the certificate.